src/main/features/dashboard/views/claimant.njk

```diff
@@ -17,7 +17,13 @@
       <div>
         <h2 class="heading-medium">{{ t('Claim status') }}</h2>
 
-        {{ responseStatusForClaimant(claim, responseOfferPage.evaluateUri({ externalId: claim.externalId })) }}
+        {% if claim.settlementReachedAt %}
```

Comment:
This block (lines 20-24) is repeated 4 times in the PR. Perhaps building this URL can be moved to a component and re-used?